### PR TITLE
Fixes nightly failures due to package availability

### PR DESCRIPTION
**Pull request by Andrewgdewar**
#### What are the changes introduced in this pull request?
Fixes nightly failures due to package availability

#### What are the testing steps for this pull request?
navigate to the Contentview index, hover over a last task: 
Screen Shot 2021-11-18 at 9 36 01 AM

### CODE REVIEW

**Comment by evgeni** [comment]
This drops the `moment` dependency, so 👍

Whether the result is the same I'll leave to someone who understands JS ;)

**Review by sjha4 — APPROVED** [review]
Tooltip looks good. Same date display as before. Ack 👍🏽

**Comment by evgeni** [comment]
Thank you two for the super quick turnaround!

File: webpack/scenes/ContentViews/Details/Repositories/__tests__/contentViewAddRemove.test.js [hunk 1]
```diff
@@ -70,7 +72,7 @@ test('Can add repositories', async (done) => {
     .put(cvDetailsPath, cvAddparams)
     .reply(200, cvRepoAddData);
 
-  const cvDetailScope = nockInstance
+  const cvDetailScope = nockInstanced
```
Comment (jeremylenz) [inline webpack/scenes/ContentViews/Details/Repositories/__tests__/contentViewAddRemove.test.js:75]: typo :)

**Review by jeremylenz — CHANGES_REQUESTED** [review]
LGTM except for the below :)

File: webpack/scenes/ContentViews/Details/Repositories/__tests__/contentViewAddRemove.test.js (hunk 1 above)
Comment (Andrewgdewar) [reply webpack/scenes/ContentViews/Details/Repositories/__tests__/contentViewAddRemove.test.js:75]: must have hit delete by accident, fixing!

**Review by jeremylenz — APPROVED** [review]

**Pull request merged**